tests/cases/fourslash/unusedImportsFS_entireImportDeclaration.ts

```diff
@@ -0,0 +1,10 @@
+/// <reference path='fourslash.ts' />
+
+// @noUnusedLocals: true
+// @Filename: /a.ts
+////import a, { b } from "mod";
+
+verify.codeFix({
+    description: "Remove declaration for: 'mod'",
```

Comment:
This wording is confusing.  What about something like "Remove import from 'mod'"?